.gitignore

```diff
@@ -12,3 +12,6 @@ pom.xml.asc
 .nrepl-port
 .cpcache/
 *.iml
+.clj-kondo/.cache
+.lsp/.cache
+.calva/output-window/output.calva-repl
```

Comment:
This should be defined in your own .gitignore

Reply:
Done see latest commit and earlier response.